dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/deletion/DeletionHandler.java

```diff
@@ -389,6 +390,15 @@ public void deleteDataEntryForm( DataEntryForm form )
     {
     }
 
+    public String allowDeleteValidationRuleNotificationTemplate( ValidationNotificationTemplate vrnt )

```

Comment:
We need to delete associated notification templates when the validation rule is deleted.